dev_utils/compose-no-tls.yml

```diff
@@ -41,19 +41,25 @@ services:
       timeout: 20s
       retries: 3
   s3:
-    command: server /data
+    command: server /data --console-address ":9001"
     container_name: s3
     environment:
-      - MINIO_ACCESS_KEY=access
-      - MINIO_SECRET_KEY=secretkey
+      - MINIO_NOTIFY_ELASTICSEARCH_ENABLE=on
+      - MINIO_NOTIFY_ELASTICSEARCH_URL=http://elasticsearch:9200
+      - MINIO_NOTIFY_ELASTICSEARCH_INDEX=minio
+      - MINIO_NOTIFY_ELASTICSEARCH_FORMAT=namespace
```

Comment:
Why are these needed?

Reply:
They are not actually, local testing leftovers. Removing them